ValveResourceFormat/Resource/ResourceTypes/ModelData/Attachments/Attachment.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections;
+using ValveResourceFormat.Serialization;
+using ValveResourceFormat.Serialization.KeyValues;
+
+namespace ValveResourceFormat.ResourceTypes.ModelData.Attachments
+{
+    public class Attachment : IEnumerable<Attachment.Influence>
+    {
+        public struct Influence
+        {
+            public string Name { get; init; }
+            public Vector3 Offset { get; init; }
+            public Quaternion Rotation { get; init; }
+            public double Weight { get; init; }
+        }
```

Comment:
```suggestion
        public record struct Influence(string Name, Quaternion Rotation, Vector3 Offset, float Weight, bool RootTransform);
```
Weight is a float in engine. Kv3 storing as double is misleading. Also is RootTransform some legacy parameter that you didn't want to keep?

Reply:
I'm not sure what the purpose of RootTransform is. It's set to true if the influence is the root transform, but in those cases, the influence name is also an empty string. It was missing from old .vmesh files.

Reply:
Okay then, if it's missing from old meshes and we make no use of for exporting/rendering then it can be ignored at this point.